### PR TITLE
Updates for 100.10 (#68)

### PR DESCRIPTION
**Pull request by nCastle1**
* Removes support for iOS 12
* Update RELEASE.md
* Bump to 100.10

### CODE REVIEW

**Comment by nCastle1** [comment]
Fully tested, everything working as expected.

File: docs/README.md [hunk 1]
```diff
@@ -307,7 +307,7 @@ The app offers the option to set a Home Location. This location is used to pre-p
...
+Once search is complete and a feature is selected on the map, the route icon appears on the bottom card. Tapping that navigates the app to the routing page, where user is prompted to add a Start and End location. The End location is always pre-populated with the feature selected on the map. The Start location is only pre-populated if Location Services is enabled or if the user has a Home Location set. When the user hits Route Me, the app navigates back to the map and the route is generated using the TransportationNetwork that was packaged in the mmpk. Read more about [routing and how to create routes using transportation networks](https://developers.arcgis.com/documentation/mapping-apis-and-location-services/routing/routing/).
```
Comment (esreli) [inline docs/README.md:310]: I scratched my head reading `[...]/routing/routing/` at first but after checking for my self it looks good to me!

**Review by esreli — CHANGES_REQUESTED** [review]
Because it would be difficult for me to check myself, I'm trusting your work here that the app version has been updated in all locations. I have a few small suggestions but otherwise this looks ready to go 👍

**Review by esreli — APPROVED** [review]
👍

**Pull request merged**